bundles/org.openhab.binding.serial/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -76,15 +76,119 @@
 			</parameter>
 			<parameter name="charset" type="text">
 				<advanced>true</advanced>
-				<label>Charset</label>
-				<description>The charset to use for converting between bytes and string (e.g. UTF-8, ISO-8859-1)</description>
+				<label>Charset or HEX</label>
```

Comment:
Think this should remain `Charset`  as HEX is a value of the parameter maccording to the docs.

Reply:
Done